proto/profile.proto

```diff
@@ -115,6 +115,24 @@ message Sample {
   // label includes additional context for this sample. It can include
   // things like a thread id, allocation size, etc
   repeated Label label = 3;
+  // Breakdown can hold the individual timestamps and values for a sample that
+  // has aggregated values. The type and unit of each breakdown is defined by
+  // the corresponding entry in Profile.sample_type. It's acceptable to have
+  // less, but not more, breakdown entries than Profile.sample_type entries.
+  repeated Breakdown breakdown = 4;
```

Comment:
Christophe had concerns about the size implication of having one `Breakdown` message per sample type for the case where the data is sparse, i.e. there is no breakdown information available for some of the sample types. My thinking is this:

In the worst case each empty breakdown message will take up 2 bytes in size (1 byte for the field number and type info, and one byte for the length of the message which is `0`).

This might not seem ideal, but I expect that a typical Sample will be at least ~50 bytes (we could analyze some data to confirm), so the having an empty breakdown is only a 4% (2/50) increase in size.

Typically I'd expect that it's always the same sample types that have empty breakdowns (because they don't support the feature), so one could eliminate that cost completely by ordering the sample types without breakdown last and only having breakdowns for the first few sample types that support it.